client/js/table-view.js

```diff
@@ -74,10 +76,50 @@ class TableView{
         this.formulaBarEl.addEventListener('keyup', this.handleFormulaBarChange.bind(this));
     }
 
+    renderSum(collie, total) {
+        const whichColumn = document.querySelector(`#sum-${collie}`);
+        whichColumn.textContent = total;
+    }
+    
+    sumOf(arr) {
+        return arr.reduce(((total, x) => total+=x),0);
```

Comment:
This should be `total + x` not `total += x`.

You're returning the new total, so you don't need the equal sign.  The assignment part happens outside of your control, within the implementation of `reduce`.